loadModelsFrom(String...)

taking several paths

Better to give it another name or it seems to give problems when executed e.g. fro ma servlet:

```
Caused by: java.lang.NoSuchMethodError: 'void edelta.lib.EdeltaVersionMigrator.loadModelsFrom(java.lang.String)'
	at edelta.example.migration.personlist.migrator.PersonsModelMigrator.execute(PersonsModelMigrator.java:25)
	at it.gssi.edelta.migrationservice.api.MigrationController.modelmigration(MigrationController.java:49)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:258)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:191)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:118)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:986)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:891)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1089)
```

--- a/edelta.parent/edelta.lib.tests/src/edelta/lib/tests/EdeltaVersionMigratorTest.java
+++ b/edelta.parent/edelta.lib.tests/src/edelta/lib/tests/EdeltaVersionMigratorTest.java
@@ -270,7 +270,7 @@ class EdeltaVersionMigratorTest {
 		versionMigrator.loadEcore(TESTDATA + subdir + METAMODELS + "v3/" + PERSON_LIST_ECORE);
 		versionMigrator.loadEcore(TESTDATA + subdir + METAMODELS + "v2/" + MY_ECORE);
 		// load the models to check for migration
-		versionMigrator.loadModelsFrom(
+		versionMigrator.loadModelsFromPaths(
 				OUTPUT + outputSubdir + "path1",
 				OUTPUT + outputSubdir + "path2");
 

--- a/edelta.parent/edelta.lib/src/edelta/lib/EdeltaVersionMigrator.java
+++ b/edelta.parent/edelta.lib/src/edelta/lib/EdeltaVersionMigrator.java
@@ -163,6 +163,26 @@ public class EdeltaVersionMigrator {
 	}
 
 	/**
+	 * Load all models in the given path (possibly recursively in subdirectories),
+	 * using the configured file extensions, by default ".xmi" files (see
+	 * {@link #addModelFileExtension(String)}).
+	 * 
+	 * @param path the path to scan for model files, e.g., "path/to/models"
+	 * @throws IOException
+	 */
+	public void loadModelsFrom(String path) throws IOException {
+		try (var stream = Files.walk(Paths.get(path))) {
+			stream
+				.filter(file -> !Files.isDirectory(file))
+				.filter(file -> {
+					var fileToString = file.toString();
+					return modelExtensions.stream().anyMatch(fileToString::endsWith);
+				})
+				.forEach(file -> loadModel(file.toString()));
+		}
+	}
+
+	/**
 	 * Load all models in the given paths (possibly recursively in subdirectories),
 	 * using the configured file extensions, by default ".xmi" files (see
 	 * {@link #addModelFileExtension(String)}).
@@ -170,17 +190,9 @@ public class EdeltaVersionMigrator {
 	 * @param paths one or more paths to scan for model files
 	 * @throws IOException
 	 */
-	public void loadModelsFrom(String... paths) throws IOException {
+	public void loadModelsFromPaths(String... paths) throws IOException {
 		for (var path : paths) {
-			try (var stream = Files.walk(Paths.get(path))) {
-				stream
-					.filter(file -> !Files.isDirectory(file))
-					.filter(file -> {
-						var fileToString = file.toString();
-						return modelExtensions.stream().anyMatch(fileToString::endsWith);
-					})
-					.forEach(file -> loadModel(file.toString()));
-			}
+			loadModelsFrom(path);
 		}
 	}
 

